lib/ui/src/components/sidebar/__tests__/data.test.ts

```diff
@@ -137,77 +104,27 @@ describe('collapse all stories', () => {
   it('collapses docs-only stories', () => {
     const hasDocsOnly: StoriesHash = {
       ...stories,
-      a1: { ...a1, parameters: { ...a1.parameters, ...docsOnly.parameters } },
+      a1: {
+        id: 'a1',
+        name: 'a1',
+        title: 'a',
+        depth: 2,
+        type: 'docs',
+        parent: 'a',
+        importPath: './a.js',
+      },
     };
 
     const collapsed = collapseAllStories(hasDocsOnly);
 
     expect(collapsed.a1).toEqual({
       id: 'a1',
       name: 'a',
-      kind: 'a',
+      title: 'a',
       depth: 1,
-      isComponent: true,
-      isLeaf: true,
-      isRoot: false,
+      type: 'docs',
       parent: 'root',
-      children: [],
-      args: {},
-    });
-  });
-
-  it('collapses mixtures of leaf and non-leaf children', () => {
```

Comment:
This tested an impossible situation -- you cannot have a story as a child of a root. Perhaps there was something legitimate this was trying to test, I am not sure. Aiming to revisit `DOCS_MODE` later.

Reply:
It looks like this was possible in the previous chromatic stories for sidebar & this has regressed (or improved?) with the changes in this PR.